docs/how-to/configure-cephfs-mirroring.rst

```diff
@@ -0,0 +1,121 @@
+==================================
+Configure CephFS replication
+==================================
+
+CephFS is a POSIX-compliant file system over Ceph's distributed storage layer consumed by various cloud workloads.
+For improved resiliency and disaster recovery, it is often desirable to replicate CephFS data to a remote cluster.
+
+MicroCeph replication framework supports enabling replication for CephFS workloads to a remote MicroCeph cluster.
+
+An operator can enable this on any subvolume or a directory path relative to the parent volume.
+
+Prerequisites
+--------------
+1. A primary and a secondary MicroCeph cluster, for example named ``primary_cluster`` and ``secondary_cluster``.
+2. Both primary_cluster and secondary_cluster have imported exchanged cluster tokens. Refer to :doc:`import remote <./import-remote-cluster>`.
+3. Both clusters have an active cephfs volume called ``vol``.
+4. Both clusters have exactly one ``cephfs-mirror`` daemon enabled. Refer to :doc:`enable cephfs-mirror <./enable-service-instances>`.
```

Comment:
- Here we can link to the exact anchor: https://canonical-microceph--642.com.readthedocs.build/642/how-to/enable-service-instances/#enable-cephfs-mirror-service
- Also, I think we should improve the readability of the sentence, e.g. Refer to our how-to guide for enabling the `cephfs-mirror` service (link text from enabling).